### PR TITLE
Add more about the OceanDB & Osmosis drivers in the "Components" page

### PR DESCRIPTION
**Pull request by ttmc**
as suggested in [a review of pull request](https://github.com/oceanprotocol/docs/pull/186#pullrequestreview-217843158) where I added a section about how to "Develop a Service Integration Driver or Plugin".

### CODE REVIEW

File: content/concepts/components.md [hunk 1]
```diff
@@ -7,10 +7,16 @@ Before reading this page, you should understand some [Ocean-specific terminology
 
 ## Aquarius
 
-Marketplaces run Aquarius to store and manage metadata about the [assets](/concepts/terminology/#asset-or-data-asset) available in their marketplace. It provides an HTTP API for interacting with an off-chain database ([OceanDB](/concepts/components/#oceandb-drivers)).
+Marketplaces run Aquarius to store and manage metadata about the [assets](/concepts/terminology/#asset-or-data-asset) available in their marketplace. It provides an HTTP API for interacting with an off-chain database (sometimes called "OceanDB").
 
 <repo name="aquarius"></repo>
 
+Aquarius supports several options for the off-chain database, including MongoDB, Elasticsearch and BigchainDB. One can add support for another off-chain database by creating a new driver similar to the existing OceanDB drivers.
```
Comment (kremalicious) [inline content/concepts/components.md]: would structure this with a subheading `### OceanDB`, and further down `### Osmosis` before the osmosis repo list text. Increases ability to scan page quickly and also makes it show up in TOC

File: content/concepts/components.md [hunk 2]
```diff
@@ -7,10 +7,16 @@ Before reading this page, you should understand some [Ocean-specific terminology
...
+
+<repo name="oceandb-mongodb-driver"></repo>
+<repo name="oceandb-bigchaindb-driver"></repo>
+<repo name="oceandb-elasticsearch-driver"></repo>
+
```
Comment (kremalicious) [inline content/concepts/components.md]: we should also link to the high-level interface, or is that not used anymore? 

Could just keep the existing text maybe

```
These drivers are extended from a high-level module implementing OceanDB functions. You can create your own plugins based on its abstracted interface:
	
<repo name="oceandb-driver-interface"></repo>
```

File: content/concepts/components.md (hunk 1 above)
Comment (ttmc) [reply content/concepts/components.md]: Okay I will add subheadings.

File: content/concepts/components.md (hunk 2 above)
Comment (ttmc) [reply content/concepts/components.md]: It _is_ still used but this page is supposed to be a high-level overview, not a detailed technical manual about how to create drivers. Anyone looking at the existing drivers will quickly learn the pattern; there's no need to elaborate here.

**Pull request merged**